### ISSUE TITLE
SW3: Unstacked view

### ISSUE THREAD

**Issue by frostburn** [xenharmonic-devs/scale-workshop#541]
Implicit `stack()` as the final line.
We could get adventurous here and replace the tuning table with a circle in this view.

**Comment by Andreya-Autumn** [comment]
Yeah this is a great idea. And very much connected to the question of "1/1 always first" or not. The situation in which I'm considering intervals both above and below the 1/1 is a situation in which I don't care for equave equivalences. Is a spontaneous thought.
